shared/clinical-code-sets/conditions/diabetic-neuropathy/1/README.md

```diff
@@ -0,0 +1,16 @@
+# Diabetic neuropathy
+
+Any code indicating a diagnosis of diabetic neuropathy, or any kind of neuropathy caused by diabetes.
+
+Code set was built from searching the SDE clusters table for terms that include both '%diabet%' and '%neuropath%'.
+
+## Prevalence log
+
+By examining the prevalence of codes (number of patients with the code in their record) broken down by clinical system, we can attempt to validate the clinical code sets and the reporting of the conditions. Here is a log for this code set. The prevalence range `0.58% - 1.53%` suggests that this code set is potentially missing some codes.
+
+
+| Date       | Practice system | Population | Patients from ID | Patient from code |
+| ---------- | --------------- | ---------- | ---------------: | ----------------: |
+| 2024-09-09 | EMIS | 2539735 | 23652 (0.931%) | 23652 (0.931%) | 
```

Comment:
Based on the meeting yesterday it sounds like our EMIS/TPP/Vision lookup is out of date. We should update that at some point.